addons/payment_stripe/views/payment_templates.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="checkout" inherit_id="payment.checkout">
+        <xpath expr="." position="inside">
+            <t t-call="payment_stripe.stripe_assets"/>
+        </xpath>
+    </template>
+
+    <template id="manage" inherit_id="payment.manage">
+        <xpath expr="." position="inside">
+            <t t-call="payment_stripe.stripe_assets"/>
+        </xpath>
+    </template>
+
+    <template id="stripe_assets" name="stripe assets">
+        <script type="text/javascript" src="https://js.stripe.com/v3/"></script>
+    </template>
```

Comment:
No need to have a brand new template just for that. Factoring code is a good practice but it's overkill here.

Reply:
I created this template to avoid changes all over the code in the future (e.g. if we need to switch to v4). I understand now that in this case it's overkill since it's only two lines to change in the same file.